pgmpy/tests/test_models/test_ClusterGraph.py

```diff
@@ -173,5 +176,38 @@ def test_check_model2(self):
         self.graph.add_factors(phi3)
         self.assertTrue(self.graph.check_model())
 
+    def test_copy(self):
+        self.graph.add_edges_from([[('a', 'b'), ('b', 'c')]])
+        phi1 = Factor(['a', 'b'], [2, 2], np.random.rand(4))
+        phi2 = Factor(['b', 'c'], [2, 2], np.random.rand(4))
+        self.graph.add_factors(phi1, phi2)
+        graph_copy = self.graph.copy()
+        self.assertIsInstance(graph_copy, ClusterGraph)
+        self.assertIsNot(self.graph, graph_copy)
```

Comment:
@Pratik151 This is wrong. This will always pass since `ClusterGraph` has no `__eq__` method.